contributed_definitions/NXmpes.nxdl.xml

```diff
@@ -128,20 +135,27 @@
                 <field name="fast_axes" type="NX_CHAR" recommended="true"/>
                 <field name="slow_axes" recommended="true"/>
                 <group type="NXcollectioncolumn">
+                    <!--
+                        TODO: Comment from Anders Frisk on proposal page
+                        What is the definition of a collection column?
+
+                        Optional constant settings (like lens focusing parameters on the sample: position_x, position_y, working_distance)
+                    -->
                     <field name="scheme">
                         <doc>
                              Scheme of the electron collection column.
                         </doc>
                         <enumeration>
                             <item value="Standard"/>
                             <item value="Angular dispersive"/>
+                            <item value="Spatial dispersive"/>
```

Comment:
This was suggested to add by Anders Frisk on the proposal page. I don't specifically know the details of the different modes and whether spatial dispersive makes sense as an additional point here.

Reply:
It's similar to what we meant by selective area, but fine. The question is what "standard" is...

Reply:
For standard, I think I was trying to think about purely XPS detectors that have neither spatial or angular configurations, but I agree that it is slightly confusing. The list should be extended a lot I imagine.